security/s2a/internal/crypter/common.go

```diff
@@ -0,0 +1,15 @@
+package crypter
```

Comment:
I'm curious why the file name is `common.go`? Are we planning to use this interface somewhere else?

Reply:
This was my suggestion, though I'm open to changing the name. Assuming the record protocol is in a file named `record.go` (like for ALTS), then I don't think this interface should go in `record.go` because the AEAD crypter will be owned by the half connection, not the record protocol. WDYT?

Reply:
We can call this file crypter. WDYT? I'm assuming we will have several implementations of this interface in separate files in this directlry.

Reply:
Would you be opposed to `aeadcrypter.go`, to distinguish it from the package name?

(I fully admit I don't know what the usual naming conventions are in Go, so I defer to you on this one.)

Reply:
`aeadcrypter.go` works fine with me. Thanks!